pagerduty/event_orchestration_path.go

```diff
@@ -97,8 +97,17 @@ type EventOrchestrationPathCatchAll struct {
 	Actions *EventOrchestrationPathRuleActions `json:"actions,omitempty"`
 }
 
+type EventOrchestrationPathWarning struct {
+	Feature string `json:"feature"`
+	FeatureType string `json:"feature_type"`
+	Message string `json:"message"`
+	RuleId string `json:"rule_id"`
+	WarningType string `json:"warning_type"`
+}
+
 type EventOrchestrationPathPayload struct {
 	OrchestrationPath *EventOrchestrationPath `json:"orchestration_path,omitempty"`
+	Warnings []*EventOrchestrationPathWarning `json:"warnings"`
```

Comment:
Question: Should we do `omitempty` for Warnings?

Reply:
I don't think we need it because `omitempty` is used to omit the property from JSON during serialization if the corresponding struct property is empty. In our case we don't send warnings to the API so we don't serialize them.